text/processing.py

```diff
@@ -10,7 +10,12 @@ class PipelineJob(Job):
     """
 
     def __init__(
-        self, text, pipeline, zip_out=False, check_equal_length=False, mini_task=False
+        self,
+        text_file,
```

Comment:
also rename `text` to `text_file` below in lines 22 and 28 

also the run method needs to be modified